Apps/GatewayApi/test/unit/Controllers.Test/PhsaControllerTests.cs

```diff
@@ -0,0 +1,129 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.GatewayApiTests.Controllers.Test
+{
+    using System;
+    using System.Collections.Generic;
+    using DeepEqual.Syntax;
+    using HealthGateway.Common.Data.Constants;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.GatewayApi.Controllers;
+    using HealthGateway.GatewayApi.Models;
+    using HealthGateway.GatewayApi.Services;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    /// <summary>
+    /// PhsaController's Unit Tests.
+    /// </summary>
+    public class PhsaControllerTests
```

Comment:
NIT: Do we test controllers, or their underlying services?

Reply:
We often don't test (simple) controllers, but it looks like this is not uniform across all our projects.  We may wish to discuss in retro as I think we should do away with these types of tests.

Reply:
Yeah it really isn't testing much.